testdata/templates/example_compute_instance.json

```diff
@@ -13,7 +13,7 @@
         "displayDevice": {
           "enableDisplay": false
         },
-        "shieldedVmConfig": {
+        "shieldedInstanceConfig": {
```

Comment:
I'm not super familiar with these tests, but it seems like we are using these files for "golden file" testing? I'm curious how we expect them to be updated as the provider adds support for new fields.

Reply:
yeah, I hear you on this not being an ideal way of doing tests. I'm not super familiar with them either / not sure how often they'll change - except to say that, in the last 9 or so months, there were about three resource changes that only broke these tests. (There were also resource changes that caused other types of issues...) Definitely into talking about another way to handle this going forward.